bin/oref0-pump-loop.sh

```diff
@@ -98,6 +100,7 @@ main() {
                 fi
             fi
             cat /tmp/oref0-updates.txt 2>&3
+            touch /tmp/pump_loop_success
```

Comment:
Why are you adding a new pump_loop_success and checking for that instead of using pump_loop_completed?  The stuff between the two is all optional attempts to refresh infrequently-refreshed stuff, and doesn't have any bearing on whether the loop was successful or not.